### DISCUSSION TITLE
错误404-error 404

### DISCUSSION THREAD

**Discussion by Xiaochengsweet233** [spencerwooo/onedrive-vercel-index#812] [General]
I need some help，Why does my website always report errors in 404.
ちょっと助けてもらいたいのですが、どうして私のホームページはいつも404新聞の間違いですか。
我要一点帮助，为什么我的网页一直都是404报错？
![image](https://user-images.githubusercontent.com/115438134/208114880-1b932709-29d4-4eb8-9797-fb80562ca325.png)
![image](https://user-images.githubusercontent.com/115438134/208114979-2c0742d0-26fa-4013-9bbf-da96bb3add11.png)
![image](https://user-images.githubusercontent.com/115438134/208115072-cc4e1663-6f2c-45f9-998f-30f741941d19.png)
求帮助！

**Comment by wdw1007** [comment]
你试着更改环境变量里的 "REDIS_URL"
更改 "redis://default:" -> "rediss://:"

#808 也有类似内容
